### PR TITLE
Automated PR for BSD-2-Clause-NetBSD

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

**Comment by jlovejoy** [comment]
would we want to make the first line optional? I think we decided this made it a different license b/c of this, but... might want to have another look?

**Comment by bradleeedmondson** [comment]
Sorry, I don't follow. Which line?

File: src/BSD-2-Clause-NetBSD.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <urls>
+    <url>http://www.netbsd.org/about/redistribution.html#default</url>
+  </urls>
+  <notes>NetBSD adopted this 2-clause license in 2008 for code contributed to NetBSD Foundation.</notes>
+  <license>
+    <copyright>
+      <p>Copyright (c) 2008 The NetBSD Foundation, Inc. All rights reserved.</p>
+    </copyright>
+    <body>
+      <p>This code is derived from software contributed to The NetBSD Foundation by
```
Comment (jlovejoy) [inline src/BSD-2-Clause-NetBSD.xml]: should this line be optional?  I think we decided this made it a different license b/c of this, but... might want to have another look?

**Comment by jlovejoy** [comment]
discussed on Aug 4 legal call: leave as is, but this may be future feature request to be able to collapse, e.g., NetBSD, FreeBSD, BSD-2-Clause

**Pull request merged**